packages/block-library/src/audio/edit.js

```diff
@@ -143,7 +160,7 @@ class AudioEdit extends Component {
 							className="components-icon-button components-toolbar__control"
```

Comment:
Same issue with the `components-icon-button` classname. Looks like it was an existing issue.

Reply:
Seems a shame to have to duplicate the code for this IconButton. Maybe it can be extracted to a separate component in the same file? Or the rendering logic could be reworked.